services/frontend-service/src/ui/ReleaseDialog.tsx

```diff
@@ -256,34 +209,24 @@ export const CreateLockButton = (props: { applicationName?: string; environmentN
                           application: applicationName,
                           environment: environmentName,
                           lockId: randomLockId(),
-                          message: message,
+                          message: 'default-lock',
                       },
                   }
                 : {
                       $case: 'createEnvironmentLock',
                       createEnvironmentLock: {
                           environment: environmentName,
                           lockId: randomLockId(),
-                          message: message,
+                          message: 'default-lock',
```

Comment:
Maybe like this?
```suggestion
                          message: 'unspecified reason',
```

`default-lock` looks a bit like we forgot to replace an internal identifier with a human-readable text.

Reply:
technically this value doesn't matter at all. because this value will be overwritten by the `actionsWithMessages` when we try to apply the actions.
https://github.com/freiheit-com/kuberpult/blob/a090c499230c8f4dbef61c495a2d384c00b8fa71/services/frontend-service/src/ui/ActionsCart/CheckoutDialog.tsx#L184-L188

I think I can use an empty string here, but I thought to keep this as this could be helpful somehow although it doesn't affect anything.

I can change the placeholder for example? or the default value in the text field (because the initial state for `lockMessage` in the input is empty  | and the placeholder is just 'default-lock' ) ?

Reply:
The correct thing (tm) would be to have different types here.